include/wx/textctrl.h

```diff
@@ -680,6 +685,17 @@ class WXDLLIMPEXP_CORE wxTextAreaBase
     virtual wxString GetValue() const = 0;
     virtual void SetValue(const wxString& value) = 0;
 
+    // overridden implementations are required to actually handle RTF
+    virtual wxString GetRTFValue() const
+    {
+        wxFAIL_MSG("GetRTFValue() should only be used with rich, multiline controls.");
+        return wxEmptyString;
+    }
+    virtual void SetRTFValue(const wxString& WXUNUSED(val))
```

Comment:
I think this should return `bool` as it looks like there could be errors here — e.g. what if the RTF is syntactically invalid?

Reply:
The underlying function returns void:

https://developer.apple.com/documentation/appkit/nstext/1527249-replacecharactersinrange

and I've never seen an error state raised when I pass in malformed RTF. It just silently does what it can with the content it seems.  The RICHEDIT2 control behaves the same way.